### PR TITLE
improved currencies section by adding new tap to update base currency

### PR DESCRIPTION
**Pull request by uyo-obong**
Screenshot_2019-09-27_15-40-49

### CODE REVIEW

File: packages/Webkul/Core/src/Http/Controllers/ChannelController.php [hunk 1]
```diff
@@ -130,6 +130,7 @@ public function update(Request $request, $id)
         Event::fire('core.channel.update.before', $id);
 
         $data = request()->all();
+        dd($data);
```
Comment (Jamesjay4199) [inline packages/Webkul/Core/src/Http/Controllers/ChannelController.php]: Remove this @uyo-obong

File: packages/Webkul/Core/src/Http/Controllers/BaseCurrencyController.php [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+            $data['receives_notification'] = 0;
+        }
+        $channel = $this->channel->find($id);
+        $channel->base_currency_id = $data['base_currency_id'];
+        $channel->save();
+        $channel->currencies()->sync($data['currencies']);
+
+
+        Event::fire('core.channel.update.after', $channel);
+        $mailSetting = MailSetting::first();
```
Comment (Jamesjay4199) [inline packages/Webkul/Core/src/Http/Controllers/BaseCurrencyController.php]: Mail setting shouldn't get updated here

Comment (Jamesjay4199) [reply packages/Webkul/Core/src/Http/Controllers/BaseCurrencyController.php]: @uyo-obong don't update the mail setting here

**Comment by uyo-obong** [comment]
Pls check now, just removed that

File: packages/Webkul/Core/src/Http/Controllers/BaseCurrencyController.php (hunk 2 above)
Comment (uyo-obong) [reply packages/Webkul/Core/src/Http/Controllers/BaseCurrencyController.php]: just removed that

File: packages/Webkul/Core/src/Http/Controllers/BaseCurrencyController.php [hunk 3]
```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Webkul\Core\Http\Controllers;
+
+use App\MailSetting;
```
Comment (Jamesjay4199) [inline packages/Webkul/Core/src/Http/Controllers/BaseCurrencyController.php]: Should this still be imported?

**Comment by Jamesjay4199** [comment]
@uyo-obong, I noticed something after merging the PR.
Can you make a follow-up PR to address the comment? 
Thanks.

**Comment by uyo-obong** [comment]
I will check it out

**Pull request merged**